src/asciidoc/docs/guidelines/persistSessionData.adoc

```diff
@@ -0,0 +1,64 @@
+= Persisting data across sessions
+
+:page-slug: persist-session-data
+:page-description: How to persist data that doesn't expire with the session.
+
+Some templates need to store data about the user's session, such as a user's name or game score.
+You can store data temporarily using `window.sessionStorage`.
+However, data stored in this way expires when the session ends.
+If you need to persist data across sessions, `window.localStorage` can be used.
+Data stored in this way has no expiration date.
+It will be available until the browser's cache is cleared.
+
+For example, here is a link to a Koji that uses local storage to store the user's name: https://withkoji.com/~Svarog1389/chatbox[Chatbox]
+
+
```

Comment:
Remove extra blank line in source (don't use two blank lines in a row).

Reply:
Done